contract-sdk/src/guest.rs

```diff
@@ -1,26 +1,45 @@
-use alloc::{
-    string::{String, ToString},
-    vec::Vec,
-};
+use alloc::{string::ToString, vec::Vec};
 use anyhow::{bail, Result};
 use bincode::{Decode, Encode};
-use risc0_zkvm::guest::env;
 use serde::de::DeserializeOwned;
 
 use crate::{
     flatten_blobs, Blob, BlobIndex, ContractInput, Digestable, HyleOutput, Identity,
     StructuredBlob, StructuredBlobData,
 };
 
-pub type RunResult = Result<String, String>;
+pub struct GuestEnv;
+
+impl GuestEnv {
```

Comment:
could it work with something like that ?

```rust
#[cfg(feature = "risc0")]
mod env {
 ...
}
#[cfg(feature = "sp1")]
mod env {
 ...
}
```
It looks better than an empty Struct

Reply:
oui ça marche aussi, à vrai dire j'ai même pas tellement besoin de la struct ou d'un wrapper du tout